sdk/spring/README.md

```diff
@@ -19,23 +19,24 @@ You can use the following command to install jars into your local repository qui
 
 ```shell
 mvn clean install \
-  -f sdk/spring/ \
+  -Dcheckstyle.skip=true \
+  -Dcodesnippet.skip \
+  -Denforcer.skip \
+  -Djacoco.skip=true \
+  -Dmaven.javadoc.skip=true \
+  -Drevapi.skip=true \
+  -DskipTests \
+  -Dspotbugs.skip=true \
   -Pdev \
   -T 4 \
+  -ntp \
   -o \
-  -Dcheckstyle.skip \
-  -Dcodesnippet.skip \
-  -Denforcer.skip \
-  -Djacoco.skip \
-  -Dmaven.javadoc.skip \
-  -Dmaven.test.skip \
```

Comment:
`maven.test.skip` can not be used because we need to build test-jar for spring-messaging-core.

Reply:
`Dmaven.test.skip=true` will skip compiling the tests.

Refs: https://maven.apache.org/surefire/maven-surefire-plugin/examples/skipping-tests.html

Reply:
but why do we need to build the test jar for users' local development?

Reply:
[spring-messaging-azure-eventhubs](https://github.com/Azure/azure-sdk-for-java/blob/14b39e6219fc3ef1ea02367de1e636d72f6f803f/sdk/spring/spring-messaging-azure-eventhubs/pom.xml#L46) depends on the test-jar.